content/tutorials/r_setup_codingclub_session/index.Rmd

```diff
@@ -31,46 +31,93 @@ remotes::install_github("inbo/inborutils")
+    │   └── YYYYMMDD_challenges.r
+└── data
+│   ├── YYYYMMDD
+|   │   ├── YYYYYMMDD_filename1.txt
+|   │   ├── YYYYYMMDD_filename2.txt   # if more than one dataset is used
+    │   └── ...
+```
+
+If you want to get the coding club materials for a past date, use the "YYYYMMDD" format, e.g. for material of August 25, 2020: 
 
+```{r change_date}
+setup_codingclub_session("20200225")
+```
 
-If you want to get the coding club materials for a past date, use the "YYYYMMDD" format, e.g.: 
+The subfolders `./src/20200225` and `./data/20200225` are automatically created and files downloaded if not present:
 
-```{r}
-setup_codingclub_session("20200825")
 ```
+├── src
+    ├── 20200225
+    │   ├── 20200225_create_messy_project.r
+    │   ├── 20200225_challenges.r
+    │   └── 20200225_challenges_solutions.r
+└── data
+│   ├── 20200225
+|   │   └── 20200225_urban_gaia_policy.txt
+```
+
+This function is also interactive: before downloading starts a message appears on screen showing the URL of the files to download and the absolute path where R scripts and data will be downloaded, followed by the question: `Do you want to continue? (Y/n)` as shown below:
+
+![example_message](figures/example_message_before_download.png)
+
+If everything is ok, enter `Y` (`y` works as well, but it's a kind of a secret...)
 
-If your folders are not named `src` or `data` (although we would recommend you name them as such) but for example `scripts` and `data_codingclub` you can specify this:
+If your folders are not named `src` or `data` (although we recommend to name them as such) but for example `scripts` and `data_codingclub` you can specify them:
 
-```{r}
-setup_codingclub_session("20200326",
+```{r other_subfolders}
+setup_codingclub_session("20200225",
                          src_rel_path = "scripts",
                          data_rel_path = "data_codingclub")
 
 ```
+
+```
+├── scripts
+    ├── 20200225
+    │   ├── 20200225_create_messy_project.r
+    │   ├── 20200225_challenges.r
+    │   └── 20200225_challenges_solutions.r
+└── data_codingclub
+│   ├── 20200225
+|   │   └── 20200225_urban_gaia_policy.txt
+```
+
+To make the function even more flexible, you can modify the root directory although you shouldn't need to do it while working within a RStudio project. For example, if you want to move the root to the parent directory:
+
+```{r other_root_dir}
+setup_codingclub_session(root_dir = "../")
+```
+
+Last but not least, you can always check examples and documentation [online](https://inbo.github.io/inborutils/reference/setup_codingclub_session.html) or via R console:
+
+```{r help}
+?setup_codingclub_session
+```
+
+Doesn't `setup_codingclub_session` work as expected? Do you think there is a bug? Or do you have some ideas to improve the function? Raise your hands! Write an [issue](https://github.com/inbo/inborutils/issues) or contact one of the [INBO coding club organizers](https://inbo.github.io/coding-club/). We will be happy to help you.
```

Comment:
```suggestion
Doesn't `setup_codingclub_session()` work as expected?
Do you think there is a bug?
Or do you have some ideas to improve the function?
Raise your hands!
Write an [issue](https://github.com/inbo/inborutils/issues) or contact one of the [INBO coding club organizers](https://inbo.github.io/coding-club/).
We will be happy to help you.
```

Reply:
Done in https://github.com/inbo/tutorials/pull/206/commits/684e3a179397784b623376df97f2962615ca778a and https://github.com/inbo/tutorials/pull/206/commits/6fa721b714773b8b1e60e86eed7c4d2f71208a3d

I also removed the s in _Raise your hands_ as it is wrong: you raise one hand to ask something, you raise both to party :smile: